service_providers_controller.rb

```diff
@@ -194,40 +178,18 @@ def service_provider_params
                   :state_id,
                   :city_id,
                   :bghex,
-                  {:category_ids => []},
-                  {:subcategory_ids => []}
-          )
-    else
-      params.require(:service_provider).
-          permit( :name,
-                  :mission,
-                  :online_tool,
-                  :short_bio,
-                  :image,
-                  :street1,
-                  :street2,
-                  :city,
-                  :state,
-                  :zip_code,
-                  :website,
-                  :contact_person,
-                  :gender,
-                  :contact_email,
-                  :phone,
-                  :fax,
-                  :cost,
-                  :orgtype,
-                  :facebook,
-                  :instagram,
-                  :twitter,
-                  :youtube,
-                  :state_id,
-                  :city_id,
-                  :bghex,
-                  {:subcategory_ids => []},
-                  {:category_ids => []}
+                  { :subcategory_ids => [] },
+                  { :category_ids => [] }
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.